.eslintrc.cjs

```diff
@@ -0,0 +1,11 @@
+module.exports= {
+    "env": {
+        "browser": false,
+        "es2021": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+  }
```

Comment:
As we go on we will add more configuration to this also @mxhdiqaim can take a look